### PR TITLE
configurable lock duration and eth amount in nearbridge contract

### PR DESCRIPTION
**Pull request by ailisp**
nearbridge contract  constructor now takes 3 arguments (add lock duration and lock eth amount)

### CODE REVIEW

**Comment by ailisp** [comment]
^ nearbridge test actually pass, but run same test with coverage takes forever, didn't figure out why

File: libs-sol/nearbridge/contracts/NearBridge.sol [hunk 1]
```diff
@@ -31,11 +31,13 @@ contract NearBridge is INearBridge {
         mapping(uint256 => BlockProducer) next_bps;
     }
 
-    uint256 constant public LOCK_ETH_AMOUNT = 1 ether;
-//    uint256 constant public LOCK_DURATION = 1 hours;
-    uint256 constant public LOCK_DURATION = 10 seconds;
+    uint256 constant public DEFAULT_LOCK_ETH_AMOUNT = 1 ether;
```
Comment (nearmax) [inline libs-sol/nearbridge/contracts/NearBridge.sol:34]: `DEFAULT_*` are not used.

File: libs-sol/nearbridge/contracts/NearBridge.sol [hunk 2]
```diff
@@ -31,11 +31,13 @@ contract NearBridge is INearBridge {
...
+//    uint256 constant public DEFAULT_LOCK_DURATION = 1 hours;
+    uint256 constant public DEFAULT_LOCK_DURATION = 10 seconds;
 
     bool public initialized;
+    uint256 public lock_eth_amount;
+    uint256 public lock_duration;
```
Comment (nearmax) [inline libs-sol/nearbridge/contracts/NearBridge.sol:40]: Is it in seconds?

File: libs-sol/nearbridge/contracts/NearBridge.sol [hunk 3]
```diff
@@ -31,11 +31,13 @@ contract NearBridge is INearBridge {
...
 
-    uint256 constant public LOCK_ETH_AMOUNT = 1 ether;
-//    uint256 constant public LOCK_DURATION = 1 hours;
-    uint256 constant public LOCK_DURATION = 10 seconds;
+    uint256 constant public DEFAULT_LOCK_ETH_AMOUNT = 1 ether;
+//    uint256 constant public DEFAULT_LOCK_DURATION = 1 hours;
+    uint256 constant public DEFAULT_LOCK_DURATION = 10 seconds;
 
     bool public initialized;
+    uint256 public lock_eth_amount;
```
Comment (nearmax) [inline libs-sol/nearbridge/contracts/NearBridge.sol:39]: Is it in Eth or Wei?

Comment (ailisp) [reply libs-sol/nearbridge/contracts/NearBridge.sol:39]: it's in wei, common practice in solidity is amount always in wei, and use ether is suffixed by the unit: `1 ether`, `ether` is then compiled to `* 1e18`

File: libs-sol/nearbridge/contracts/NearBridge.sol (hunk 2 above)
Comment (ailisp) [reply libs-sol/nearbridge/contracts/NearBridge.sol:40]: Yes, also common practice to be in second: https://solidity.readthedocs.io/en/v0.4.24/units-and-global-variables.html#time-units

File: libs-sol/nearbridge/contracts/NearBridge.sol (hunk 1 above)
Comment (ailisp) [reply libs-sol/nearbridge/contracts/NearBridge.sol]: removed

**Pull request merged**